grakn-kb/src/test/java/ai/grakn/kb/internal/GraknTxTest.java

```diff
@@ -131,6 +131,11 @@ public void whenGettingSubTypesFromRootMeta_IncludeAllTypes(){
     }
 
     @Test
+    public void whenGettingTheShardingThreshold_TheCorrectValueIsReturned(){
+        assertEquals(100000L, tx.admin().shardingThreshold());
```

Comment:
so we are not using the config in grakn.properties?

Reply:
Technically this IS the config from grakn.properties. Just in a very roundabout way